<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="contentsWidget">
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="pagesWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="general">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Units:</string>
               </property>
               <property name="buddy">
                <cstring>unitsCombo</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="unitsCombo">
               <property name="currentText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Ground reference</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QRadioButton" name="autoReferenceButton">
                <property name="text">
                 <string>Automatic</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="fixedReferenceButton">
                <property name="text">
                 <string>Fixed</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="fixedReferenceEdit"/>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="fixedReferenceUnits">
                <property name="text">
                 <string>m</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>200</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="aerodynamics">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Mass of jumper:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>kg</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Planform area:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>m^2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="areaEdit"/>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="massEdit"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="minDragEdit"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="minLiftEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Minimum lift:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Maximum lift:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Minimum drag:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="maxLDEdit"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Maximum L/D:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>172</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="maxLiftEdit"/>
           </item>
           <item row="7" column="0" colspan="3">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Simulation time:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>s</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSpinBox" name="simTimeSpinBox">
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>4</number>
             </property>
             <property name="value">
              <number>120</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="plots">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="plotTable">
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Line thickness:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineThicknessEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>pixels</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="wind">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Wind speed:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="windSpeedEdit"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="windUnitsLabel">
             <property name="text">
              <string>units</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Wind direction:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="windDirectionEdit"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>deg</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>176</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
